src/pages/ShpeitoNetwork.js

```diff
@@ -86,6 +74,18 @@ function ShpeitoNetwork(props) {
     });
   }
 
+  const [filter, setFilter] = useState(
```

Comment:
The shpeito network won't load. I think it is because you declare filter/setFilter before filter is used in the previous lines. Just move the declaration to earlier in the function and I think you'll be good. Thanks, Jorge 😎